File: src/main/java/com/uwetrottmann/trakt5/entities/ListEntry.java
package com.uwetrottmann.trakt5.entities;

import org.threeten.bp.OffsetDateTime;

public class ListEntry {

    public Long id;
    public Integer rank;
    public OffsetDateTime listed_at;
    public String type;
    public Movie movie;
    public Show show;
    public Episode episode;
    public Person person;

}

File: src/main/kotlin/com/redissi/trakt/entities/ListEntry.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import java.time.OffsetDateTime

@JsonClass(generateAdapter = true)
class ListEntry(
    val id: Long? = null,
    val rank: Int? = null,
    @Json(name = "listed_at")
    val listedAt: OffsetDateTime? = null,
    val type: String? = null,
    val movie: Movie? = null,
    val show: Show? = null,
    val episode: Episode? = null,
    val person: Person? = null
)